src/components/App.js

```diff
@@ -33,57 +31,33 @@ class App extends Component {
     fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
     .then(response => {
       if(response.ok) {
-        return response.json()
+        return response.json();
       } else {
-        throw Error(response.status)
+        throw Error(response.status);
       }})
       .then(movie => this.setState({ movies: [...this.state.movies],
         movieSelected: movie }))
       .catch(error => {
-        console.log(error)
+        console.log(error);
         this.setState({ movies: [], movieSelected: { },
         status: true })})
   }
 
-
-  // findMovie = (id) => {
-  //   const movie = this.state.movies.find((movie) => movie.id === id);
-  //   // set movie to be the object.
-  //   this.setState({ movies: [...this.state.movies],
-  //                   movieSelected: movie })
-  // }
-
   redirectHome = () => {
     this.setState({ movies: [...this.state.movies],
-                    movieSelected: null })
+                    movieSelected: null });
   }
 
   render() {
-
-    // conditional logic goes here-- say if the element that was clicked has an id matching one of the ids in the state, then render that movie card
     return (
       <main className='App'>
         <h1>Rancid Tomatillos</h1>
         {this.state.status && <h2>Oops, something went wrong</h2>}
-        {this.state.movieSelected ? <MovieDetailsContainer movieSelected={ this.state.movieSelected } redirectHome={ this.redirectHome} status={this.state.status}/> : <MoviesContainer movies={this.state.movies} findMovie={ this.findMovie }/> }
+        {this.state.movieSelected ?
```

Comment:
This ternary is ok, but if it gets any more complex, then I would try another strategy for conditional rendering because it will become very difficult to read.